conda_forge_tick/feedstock_parser.py

```diff
@@ -705,6 +709,8 @@ def load_feedstock_containerized(
         extra_container_args=[
             "-e",
             f"{ENV_CONDA_FORGE_ORG}={settings().conda_forge_org}",
+            "-e",
+            f"{ENV_GRAPH_GITHUB_BACKEND_REPO}={settings().graph_github_backend_repo}",
         ],
     )
```

Comment:
these additions are a hotfix until we have a solution for https://github.com/conda-forge/conda-forge-feedstock-ops/pull/49